### PR TITLE
48 cdk framework integration

### PR DESCRIPTION
**Pull request by bdemann**

### CODE REVIEW

File: examples/query/src/main.py [hunk 1]
```diff
@@ -1,5 +1,40 @@
-from kybra import query, text
+from kybra import opt, query, Record, update, text
+from typing import TypedDict
 
 @query
 def simple_query() -> text:
     return 'This is a query function'
+
```
Comment (bdemann) [inline examples/query/src/main.py:7]: Why is all of this here?

**Comment by lastmjs** [comment]
simple_user_accounts haven't been added to the tests

File: examples/query/src/main.py (hunk 1 above)
Comment (lastmjs) [reply examples/query/src/main.py:7]: None of  that should be there, it's not like that on my branch

Comment (bdemann) [reply examples/query/src/main.py:7]: Yeah, I put it there, and I know why it's there. I guess the comment would be better said "Why did I leave this in here?" and the answer would probably be I forgot to take it out...

File: kybra/compiler/kybra_generate/src/kybra_attempt/canister_methods/mod.rs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+use crate::{
```
Comment (lastmjs) [inline kybra/compiler/kybra_generate/src/kybra_attempt/canister_methods/mod.rs:1]: Should be deleted, all kybra_attempt

**Review by lastmjs — DISMISSED** [review]

**Pull request merged**